lib/node_modules/@stdlib/utils/error-to-json/lib/type.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+// MODULES //
+
+var CTORS = require( './ctors.js' );
+
+
+// MAIN //
+
+/**
+* Returns the error type.
+*
+* @private
+* @param {(Error|TypeError|SyntaxError|URIError|ReferenceError|RangeError|EvalError)} error - input error
+* @returns {string} error type
+*
+* @example
+* var err = new TypeError();
+* var out = typeName( err );
+* // returns 'TypeError'
+*/
+function typeName( error ) {
```

Comment:
We should probably add a note in the README that determining the error type only works if provided an `error` from the same realm. Otherwise, if we want this pkg to work cross-realm, we need to use `@stdlib/utils/type-of` or `@stdlib/utils/constructor-name`, which rely on either `Object.prototype.toString` (which can be faked) or the constructor `name`, which can be set.